chainConfig/stride.json

```diff
@@ -38,7 +38,7 @@
         "https://rpc.stride.silentvalidator.com",
         "https://stride.rpc.kjnodes.com"
       ],
-      "clientId": "07-tendermint-80",
+      "clientId": "25-tendermint-09",
```

Comment:
I think this value should be 

`07-tendermint-77`

Reply:
Line 50 should be:        ` "sourceChannel": "channel-9",`

Line 52 should be: `"destinationChannel": "channel-25"`